package/skelcd-control-SLES.changes

```diff
@@ -1,3 +1,7 @@
+Wed Jan 31 15:32:36 CET 2018 - aginies@suse.com
+
+- remove ifarch for yast2-vm (BSC#1059952)
```

Comment:
are you sure that it will work? This change just ensure that package will be on DVD in insts-sys, otherwise it does not need to be there.

Reply:
There is a kind of strange behaviour which lead to a situation that yast2-vm is requires by the installer and basesystem, at the end yast2-vm finish in overlap group, and then unstorted, so it is no more in the media. Sounds like this came from this ifarch.

Reply:
ok, it does not hurt, so lets try it. Does it affect also SLED and opensuse?

Reply:
sounds like the bug in the build system has been found, and we can cancel this PR, we don't need this workaround anymore. Thanks for the review.